packages/public/server/src/module/Authorization/src/Service/AuthorizationService.php

```diff
@@ -30,12 +30,15 @@ class AuthorizationService extends \ZfcRbac\Service\AuthorizationService
 {
     /**
      * @param \Rbac\Permission\PermissionInterface|string $permission
-     * @param null                                        $context
+     * @param mixed $context
+     * @param array $roles
      * @return bool
      */
-    public function isGranted($permission, $context = null)
+    public function isGranted($permission, $context = null, array $roles = null)
```

Comment:
That looks like it will cause a runtime error.

Generally: you should be super careful when placing actual types in PHP (because PHP will throw at runtime if the types don't match). This is especially dangerous for legacy code where the comment might suggest that the type is always string but in practice it may be null. I'm not sure if PHP accepts `null` as array, but I'd triple-check that before adding the type here.

Reply:
Good catch! I removed the type hint there. There is `?array` for this situation, however our php version does not support it. I forgot to remove the type hint there.